src/main/java/org/jabref/gui/maintable/MainTableDataModel.java

```diff
@@ -54,6 +54,7 @@ public MainTableDataModel(BibDatabaseContext context, PreferencesService prefere
         // We need to wrap the list since otherwise sorting in the table does not work
         entriesSorted = new SortedList<>(entriesFiltered);
         groupViewMode = preferencesService.getGroupViewMode();
+
```

Comment:
Sneaky empty line in here? Could you remove it at the PR where you are really working on that?